packages/manager/modules/vps/src/routing.js

```diff
@@ -27,6 +27,8 @@ export default /* @ngInject */ ($stateProvider) => {
           ),
       defaultPaymentMethod: /* @ngInject */ (ovhPaymentMethod) =>
         ovhPaymentMethod.getDefaultPaymentMethod(),
+      engagement: /* @ngInject */ (serviceInfo, VpsService) =>
```

Comment:
Call not handled by component ?

Reply:
engagement is required not only in the component. We need it in the VPS dashboard too, for example for the terminate pop-up